articles/model-anatomy.adoc

```diff
@@ -0,0 +1,678 @@
+include::../shared/issue-footnotes.adoc[]
+:serialization: ../serialization/serialization
+:m3: ../metametamodel/metametamodel
+:refarch: ../reference-architecture/reference-architecture
+:roadmap: ../roadmap/roadmap
+= Anatomy of a LionWeb Model
+:toc: preamble
+:toclevels: 3
+:sectnums:
+
+This article walks us through a LionWeb model:
+What it is, what's inside, how the inside looks, what the contents mean, and why they are that way.
+It is meant as an introduction to some of the details of LionWeb.
+We don't assume any prior knowledge of LionWeb; some experience with general modeling ideas is helpful.
+
+== Introduction
+The fundamental world view in LionWeb is "everything is a model".
+This means we express everything we talk about in this context in the form of models.
+
+But what is a model?
+STOP -- don't run away!
+This question is infamous in the modeling community -- but we have to answer it given our world view as stated above.
+We deliberately don't use that term in our specifications, as it's so overloaded and ambiguous.
+For our purposes, we can always replace "model" with "a bunch of nodes".
+Then the question transforms to: What is a node?
+
+[[notation]]
+== Notation used in this article and diagrams
+We use integer numbers as node ids, and short names for keys of language elements.
+We label boxes in diagrams that represent M1-level nodes with the node id and its classifier, as in `10:Person`.
+Boxes in diagrams for M2-level language elements are labelled only with the classifier, i.e. `Person`.
+Boxes that represent concepts or concept instances use `C` icon, and `@` icon for annotation (instance) boxes.
+
+[[nodes]]
+== Nodes
+A node is the fundamental building block in LionWeb.
+It consists of a unique identifier (**id**), a **classifier** to define the nature / type / class of the node, the node's **parent** node and lists of **properties**, **containments**, **references**, and **annotation instances**, respectively.
+We group a node's properties, containments, and references as the **features** of that node.
+
+Properties are simple values like `name: String`, `age: Integer` or `alive: Boolean`.
```

Comment:
Different font/ styling for LionWeb terminologies like Property, containment and References. It would also be nice to have these in bullet points or the likes:

* Properties are...
* Containment...
* References ...